packages/kmod-5.10-nvidia/kmod-5.10-nvidia.spec

```diff
@@ -71,7 +71,7 @@ Requires: %{name}-tesla(fabricmanager)
 %package open-gpu-%{tesla_major}
 Summary: NVIDIA %{tesla_major} Open GPU driver
 Version: %{tesla_ver}
-License: MIT OR GPL-2.0-only
+License: MIT OR (MIT AND GPLv2)
```

Comment:
`GPLv2` is also not an SPDX identifier, you have to pick one of `GPL-2.0-or-later` or `GPL-2.0-only`.

`GPL-2.0-or-later` is what the open driver license says but in practice - given that it's being linked to the Linux kernel - `GPL-2.0-only` is what you end up with.

So I would change this to `MIT AND GPL-2.0-only`.